dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/Sidebar.java

```diff
@@ -0,0 +1,43 @@
+package com.dotmarketing.portlets.templates.design.bean;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * It's a {@link com.dotmarketing.portlets.templates.model.Template}'s Sidebar
+ */
+public class Sidebar extends ContainerHolder {
+
+    public static String LOCATION_RIGHT = "right";
+    public static String LOCATION_LEFT = "left";
+
+    public String location;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make location a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)